src/brush.rs

```diff
@@ -40,6 +41,7 @@ impl Default for Brush {
 /// the type as `impl<Into<BrushRef>>` allows accepting types like `&LinearGradient`
 /// directly without cloning or allocating.
 #[derive(Clone, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize))]
 pub enum BrushRef<'a> {
```

Comment:
You mentioned in #16 that maybe `BushRef` shouldn't derive even `Serialize` and I'm wondering the same. If this can be only serialized and not deserialized, what is the use case for it? 🤔

However if we keep it, then perhaps `StyleRef` should get a `Serialize` derive as well?

Reply:
Ahh, I had forgotten about that bit, so I'll go ahead and remove it.  Then if someone figures out a use for it in the future it could be added then (like perhaps it can be deserialized into an owned Brush).